includes/html/pages/device/overview/toner.inc.php

```diff
@@ -4,7 +4,7 @@
 
 $graph_type = 'toner_usage';
 
-$supplies = \App\Models\Toner::query()->where('device_id', $device['device_id'])->get()->groupBy('toner_type');
+$supplies = \App\Models\PrinterSupply::query()->where('device_id', $device['device_id'])->get()->groupBy('printer_type');
```

Comment:
->groupBy('printer_type');  this doesn't make much sense.  why did you prefix the fields with printer_?

Reply:
it does makes sense since the data is grouped by it :)